articles/organizations/index.md

```diff
@@ -0,0 +1,122 @@
+---
+url: /organizations
+section: articles
+title: "Organizations - Beta"
+description: Learn about Auth0's Organizations feature, which is currently in beta.
+topics:
+  - organizations
+contentType: index
+useCase:
+  - build-an-app
+toc: true
+---
+
+# Organizations - Beta
+
+The Organizations feature represents a broad update to the Auth0 platform that allows our Business-to-Business (B2B) customers to better manage their partners and customers, and customize the ways that end-users in those organizations access their applications. Auth0 customers can use Organizations to:
+
+* manage the businesses that access their applications.
+* build administration dashboards in their products that allow those businesses to manage their organizations, invite users, and define users’ roles within organizations.
+
+## Business cases
+
+Organizations best supports customers who are building applications that end-users access in the context of the customer’s business.
```

Comment:
... in the context of _their_ business (not the customer's business). e.g. joe schmoe logs into staff0 in the context of Initech (the company that he works for).